bee-api/bee-rest-api/src/endpoints/routes/api/v1/output.rs

```diff
@@ -71,12 +58,13 @@ pub(crate) async fn output<B: StorageBackend>(
     })? {
         (Ok(response), ledger_index) => match response {
             Some(output) => {
-                let consumed_output = Fetch::<OutputId, ConsumedOutput>::fetch(&*storage, &output_id).map_err(|e| {
-                    error!("unable to fetch the output: {}", e);
-                    reject::custom(CustomRejection::ServiceUnavailable(
-                        "unable to fetch the output".to_string(),
-                    ))
-                })?;
+                let consumed_output =
+                    Fetch::<OutputId, ConsumedOutput>::fetch(&*args.storage, &output_id).map_err(|e| {
+                        error!("unable to fetch the output: {}", e);
+                        reject::custom(CustomRejection::ServiceUnavailable(
+                            "unable to fetch the output".to_string(),
+                        ))
+                    })?;
```

Comment:
I prefer using `match` rather than `map_err` if there are any side-effects:

```suggestion
                let consumed_output = match Fetch::<OutputId, ConsumedOutput>::fetch(&*args.storage, &output_id) {
                    Err(e) => {
                        error!("unable to fetch the output: {}", e);
                        return reject::custom(CustomRejection::ServiceUnavailable(
                            "unable to fetch the output".to_string(),
                        ));
                    }
                    Ok(output) => output,
                };
```